health.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 	"bufio"
 	"os"
+	"github.com/kkdai/maglev"
```

Comment:
I don't see a license for this maglev implementation.  (The README references something called `etcd` which doesn't seem to be relevant, and a "LICENSE" file that doesn't exist.)  Please make sure that code we use is licensed appropriately.